src/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf) format(truetype);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf) format(truetype);
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.input {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  width: 100%;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  outline: none;
+  font-weight: 300;
+  font-family: inherit;
+}
+
+.input_big {
+  padding-left: 62px;
+  height: 70px;
+}
```

Comment:
you could add these styles to `.input`, and only change styles for `.input_small`

Reply:
the same for other cases to avoid duplications